### DISCUSSION TITLE
Cannot import pandas in pythonnet

### DISCUSSION THREAD

**Discussion by muler415** [pythonnet/pythonnet#2223] [Q&A]
### Environment

- Pythonnet version: 3.0.1
- Python version: 3.8
- Operating System: win11
- .NET Runtime: 4.6.1

### Details

-   Describe what you were trying to get done.

  my python script only have one line - 
  
```python
import pandas as pd
```
when I exec and exception is thrown 
partially initialized module 'pyarrow' has no attribute '__version__' (most likely due to a circular import);

please help.

**Comment by sb5m** [comment]
Hi @muler415, how is your script called? Check if there are any namespace conflicts in your code or project structure that might be causing the circular import issue. This can happen if you have files with the same name as modules. For example "version.py" or "main.py".

**Comment by muler415** [comment]
I fixed this issue by installing pyarrow. pip install pyarrow. the error is gone after that.
